rafthttp/transport.go

```diff
@@ -165,10 +165,11 @@ func (t *Transport) Send(msgs []raftpb.Message) {
 		to := types.ID(m.To)
 
 		t.mu.RLock()
-		p, ok := t.peers[to]
+		p, pok := t.peers[to]
+		g, rok := t.remotes[to]
 		t.mu.RUnlock()
```

Comment:
good catch. Would like to test this code to see if we still get https://github.com/coreos/etcd/issues/4855.